caikit/core/data_model/training_status.py

```diff
@@ -40,3 +42,12 @@ def is_terminal(self):
             self.__class__.CANCELED,
             self.__class__.ERRORED,
         ]
+
+
+@dataobject(PACKAGE_COMMON)
+class TrainingInfo(DataObjectBase):
+    errors: Annotated[List[str], FieldNumber(1)]
+    # TODO: Add elements to conveying other useful information
```

Comment:
This is interesting. I think the best we can do there is add `info: JsonDict` to be an arbitrary blob